egov-custom-consumer/LOCALSETUP.md

```diff
@@ -0,0 +1,22 @@
+# Local Setup
+
+To setup the egov-custom-consumer service in your local system, clone the [Utilities repository](https://github.com/egovernments/utilities).
+
+## Dependencies
+
+### Infra Dependency
+
+- [ ] Postgres DB
+- [ ] Redis
+- [ ] Elasticsearch
+- [X] Kafka
+  - [X] Consumer
+  - [ ] Producer
+
+## Running Locally
+
+To run the egov-custom-consumer service locally, update below listed properties in `application.properties` before running the project:
+
+```ini
+egov.coexistence.hostname=#Host value of the co-existence finance erp server (ex: https://jalandhar-dev.egovernments.org)
```

Comment:
Explain what happens in case of different tenants?

Reply:
There will not be any issue in case of different tenants as we are deleting the session and access token from the redis server irrespective of the tenant id.